bleak/__init__.py

```diff
@@ -77,8 +77,21 @@
             "Requires at least Windows 10 version 0.16299 (Fall Creators Update)."
         )
 
-    from bleak.backends.dotnet.scanner import BleakScannerDotNet as BleakScanner  # noqa
-    from bleak.backends.dotnet.client import BleakClientDotNet as BleakClient  # noqa
+    # If the winrt package is installed, assume that the user has opted to use that backend
+    # instead of the pythonnet/BleakBridge implementation.
+    try:
+        from bleak.backends.winrt.scanner import (
+            BleakScannerWinRT as BleakScanner,
+        )  # noqa
```

Comment:
I think it is nice to give the specific warning/error that is being supressed

    # noqa: F401

Reply:
👍🏻